admin_panel/models.py

```diff
@@ -54,38 +55,72 @@ class Events(models.Model):
     end_date = models.DateTimeField()
     number_of_panels = models.IntegerField(blank=True, null=True)
     venue_link = models.URLField(max_length=200)
-    date_created = models.DateTimeField(auto_now_add=True)
-    date_updated = models.DateTimeField(auto_now=True)
+    date_created = models.DateTimeField()
+    date_updated = models.DateTimeField()
     total_seats = models.IntegerField()
-    organizer = models.ForeignKey(Organizer, on_delete=models.CASCADE)
+    organizer = models.ManyToManyField(Organizer)
     user = models.ManyToManyField(CustomUser)
 
     def __str__(self):
-        return str(self.events_name)
+        return str(self.event_name)
+
+
+class Panels(models.Model):
+    panel_id = models.AutoField(primary_key=True)
+    panel_no = models.IntegerField(null=True, blank=True)
+    panel_name = models.CharField(max_length=200)
+    panel_description = models.CharField(max_length=200)
+    panel_topic = models.CharField(max_length=200)
+    panel_image = models.URLField(max_length=200, blank=True, null=True)
+    panel_start_time = models.DateTimeField()
+    panel_end_time = models.DateTimeField()
+    date_created = models.DateTimeField()
+    date_updated = models.DateTimeField()
+    events = models.ForeignKey(Events, on_delete=models.CASCADE)
+    user = models.ManyToManyField(CustomUser)
+
+    def __str__(self):
+        return self.panel_name
 
 
 class TicketInfo(models.Model):
+    TICKET_TYPE_CHOICES = (
+        ("with_food", "With Food"),
+        ("without_food", "Without Food"),
+        ("online", "Online"),
+        ("guest", "GUEST"),
+    )
     ticket_id = models.AutoField(primary_key=True)
-    ticket_type = models.CharField(max_length=100)
+    ticket_type = models.CharField(max_length=100, choices=TICKET_TYPE_CHOICES)
+    ticket_type1 = models.CharField(
+        max_length=100, blank=True, null=True
+    )  # for future use
     ticket_price = models.IntegerField()
-    ticket_description = models.CharField(max_length=100)
+    ticket_image = models.URLField(max_length=200, blank=True, null=True)
+    ticket_description = models.CharField(max_length=100, blank=True, null=True)
     tickets_available = models.IntegerField()
-    tickets_sold = models.IntegerField()
-    tickets_scanned = models.IntegerField()
+    tickets_sold = models.IntegerField(blank=True, null=True)
```

Comment:
instead of make it nullable, set the default value to 0

Reply:
ok